ludwig/distributed/ddp.py

```diff
@@ -77,7 +77,17 @@ def gather_all_tensors_fn(cls) -> Optional[Callable]:
 
     @classmethod
     def get_ray_trainer_backend(cls, **kwargs) -> Optional[Any]:
-        return "torch"
+        from ray.train.torch import TorchConfig
+
+        backend = kwargs.get("backend", None)
```

Comment:
It looks like this `backend` conflicts with the current definition of backend defined in `get_trainer_kwargs` here: https://github.com/ludwig-ai/ludwig/blob/master/ludwig/backend/ray.py#L132

Am I missing something?